web-wallet/src/lib/containers/StakeContract/StakeContract.svelte

```diff
@@ -1,171 +1,171 @@
-<svelte:options immutable={true}/>
+<svelte:options immutable={true} />
 
 <script>
-	import {
-		collect,
-		getKey,
-		hasKeyValue,
-		map,
-		mapWith,
-		pick,
-		setKey,
-		when
-	} from "lamb";
-	import { mdiCloseThick } from "@mdi/js";
+  import {
+    collect,
+    getKey,
+    hasKeyValue,
+    map,
+    mapWith,
+    pick,
+    setKey,
+    when,
+  } from "lamb";
+  import { mdiCloseThick } from "@mdi/js";
 
-	import { createCurrencyFormatter } from "$lib/dusk/currency";
-	import { getLastTransactionHash } from "$lib/transactions";
-	import {
-		operationsStore,
-		settingsStore,
-		walletStore
-	} from "$lib/stores";
-	import {
-		ContractOperations,
-		ContractStatusesList,
-		Stake
-	} from "$lib/components";
-	import {
-		ErrorDetails,
-		Icon,
-		Throbber
-	} from "$lib/dusk/components";
+  import { createCurrencyFormatter } from "$lib/dusk/currency";
+  import { getLastTransactionHash } from "$lib/transactions";
+  import { operationsStore, settingsStore, walletStore } from "$lib/stores";
+  import {
+    ContractOperations,
+    ContractStatusesList,
+    Stake,
+  } from "$lib/components";
```

Comment:
The rule that adds the extra comma seems off line 18 does not have the extra comma added but on line 22 it was added. This needs to be investigated further.

Reply:
> The rule that adds the extra comma seems off line 18 does not have the extra comma added but on line 22 it was added. This needs to be investigated further.

Seems the extra comma is added only in multi line imports. Try it [here](https://prettier.io/playground/).

(we are currently using the `es5` setting for that)

Reply:
This to me is very inconsistent and confusing. We either have a comma everywhere or we do not it should be simple.

Reply:
> This, to me, is very inconsistent and confusing. We either have a comma everywhere or we do not it should be simple.

The comma-dangle rule has been deprecated from the ESlint library and we're following `es5` (as discussed with @ZER0) approach with ``. I believe the only options to changing this would be via the  `trailing-comma` setting, which is something we already discussed and decided on, or by adding [ESLint Stylistic](https://eslint.style/rules/js/comma-dangle); but I think it's a bit of an anti-pattern to go in that direction, as all the formatting should be done by Prettier now.  However, if it's not enough then I am open to discussing adding it.